templates/gin/files/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/platformsh/template-gin
+
+require (
+	github.com/gin-gonic/gin v1.4.0
+	github.com/go-sql-driver/mysql v1.4.1
+	github.com/platformsh/config-reader-go/v2 v2.0.1
+	google.golang.org/appengine v1.6.1 // indirect
```

Comment:
indirect meaning it was added automatically?  Is... Google App Engine really a library that needs to be installed when running on Psh?  Or does this automatically get omitted at compile time if we don't use it?  I have so many questions (mostly about Google making proprietary assumptions...)

Reply:
Hm, OK, it looks like that lib is much smaller than I feared, it just shares a name with Google's weak competitor to us. :smile:  Ignore this point.

Reply:
Ya when you run `go mod tidy` it finds all the indirects and adds them for you. Lots of questions on my end as well.